### DISCUSSION TITLE
Is shorter Timelapse interval possible?

### DISCUSSION THREAD

**Discussion by hwy0010** [gopro/labs#834] [Q&A] [answered]
Hi,

I'm using GP11 for driving video, normally I will use TimeWarp, but it has poor performance at night, so I use timelapse to get a brighter image, but 0.5s interval is a bit big for me, is it possible to make it 0.1s?

**Comment by dnewman-gpsw** [comment, marked as answer]
0.1s is not possible.  The fastest is 0.25s through a Labs hack.  This is using Night Lapse Video, with Auto interval and Auto shutter, but limiting the shutter with Labs to 1/4s using ```oMEXPX=4```

**Reply by hwy0010** [reply]
Thank you so much, this is exactly what I've been looking for. Another question: is there a hack for timewarp video? like a slower shutter speed.

**Reply by dnewman-gpsw** [reply]
Timewarp only works as it shoot video at 30fps, so the shutter is limited by the capture frame rate.  Timewarp's magic is it ability to dynamically select frames from the video feed.
